docs/overview.adoc

```diff
@@ -150,8 +150,8 @@ Returns a virtual graph that represents the labels and relationship-types availa
 | CALL apoc.meta.graphSample() | examines the database statistics to build the meta graph, very fast, might report extra relationships
 | CALL apoc.meta.graph | examines the database statistics to create the meta-graph, post filters extra relationships by sampling
 | CALL apoc.meta.subGraph({labels:[labels],rels:[rel-types],excludes:[label,rel-type,...]}) | examines a sample sub graph to create the meta-graph
-| CALL apoc.meta.data | examines a subset of the graph to provide a tabular meta information
-| CALL apoc.meta.schema | examines a subset of the graph to provide a map-like meta information
+| CALL apoc.meta.data({sample:{label1:<number of each elements>, label2:<number of each elements>, ...}}) | examines a subset of the graph to provide a tabular meta information, with sample config the dataset is sampling by label
```

Comment:
No we don't want to provide that per label. We just want to provide: "sample at least XXX" elements.
E.g. `sample: 1000`.

Reply:
We can grab the total counts from the count store and compute the partial skip numbers.

Reply:
ping @AngeloBusato